example/lib/main.dart

```diff
@@ -162,6 +163,155 @@ class _HtmlEditorExampleState extends State<HtmlEditorExample> {
                   print('pasted into editor');
```

Comment:
please remove it or keep it on debug only

Reply:
done